### PR TITLE
ISO20022 to MT1XX Conversions

### PR DESCRIPTION
**Pull request by NiTR0-CH4RG3R**

### CODE REVIEW

File: mapper/.gitignore [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+# Remove this if you want to commit generated sources.
+generated/
+
+# Contains configuration values used during development time.
+# See https://ballerina.io/learn/provide-values-to-configurable-variables/ for more details.
+Config.toml
+
+Dependencies.toml
+
+.vscode/
```
Comment (VimukthiRajapaksha) [inline mapper/.gitignore:15]: Missed EOF. please check other places too

File: mapper/Module.md [hunk 2]
```diff
@@ -0,0 +1,6 @@
+Prints "Hello, World!" with a main function.
```
Comment (VimukthiRajapaksha) [inline mapper/Module.md:1]: Let's add a proper explanation to the module

File: mapper/.gitignore (hunk 1 above)
Comment (NiTR0-CH4RG3R) [reply mapper/.gitignore:15]: Done

File: mapper/Module.md (hunk 2 above)
Comment (NiTR0-CH4RG3R) [reply mapper/Module.md]: Done

File: mapper/mx_record_type_map.bal [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+    result = mxRecordTypeMap[mxRecordTypeName];
+    
+    if (result is typedesc<record {}>) {
+        return result;
+    }
+
+    return error("Unsupported MX message type: " + mxRecordTypeName);
+}
+
+
```
Comment (Ashi1993) [inline mapper/mx_record_type_map.bal:76]: Remove extra lines

File: mapper/pacs_008_transform.bal [hunk 4]
```diff
@@ -0,0 +1,573 @@
+// Copyright (c) 2023, WSO2 LLC. (https://www.wso2.com).
```
Comment (Ashi1993) [inline mapper/pacs_008_transform.bal:1]: year should be 2024

File: mapper/pacs_008_transform.bal [hunk 5]
```diff
@@ -0,0 +1,573 @@
...
+                MT20, MT13C, MT23B, MT23E, MT26T, MT32A, MT33B, MT36, MT50A, MT50F, MT50K, MT51A, MT52A, MT52D, 
+                MT53A, MT53B, MT54A, MT54B, MT54D, MT55A, MT55B, MT55D, MT56A, MT56C, MT56D, MT57A, MT57B, MT57C, MT59, MT59A, MT59F, MT71A, 
+                MT71F, MT71G, MT72, MT77B, MT77T
+            };
+        }
+    }
+
+    return error("Error occurred while creating MT103 block4");
+
+}
```
Comment (Ashi1993) [inline mapper/pacs_008_transform.bal:573]: Add EoF line for all the files

Comment (NiTR0-CH4RG3R) [reply mapper/pacs_008_transform.bal]: Done in https://github.com/ballerina-platform/module-ballerinax-financial.iso20022-to-swiftmt/pull/1/commits/15280bae5517d16c94776c14140deae3301141ca

File: mapper/mx_record_type_map.bal (hunk 3 above)
Comment (NiTR0-CH4RG3R) [reply mapper/mx_record_type_map.bal:76]: Done in https://github.com/ballerina-platform/module-ballerinax-financial.iso20022-to-swiftmt/pull/1/commits/15280bae5517d16c94776c14140deae3301141ca

**Review by VimukthiRajapaksha — APPROVED** [review]

**Review by Ashi1993 — APPROVED** [review]

**Pull request merged**